### DISCUSSION TITLE
Zwift on a 4k TV in my pain cave (hardware details in post)

### DISCUSSION THREAD

**Discussion by urge4code** [netbrain/zwift#82] [Show and tell]
Got it running on 4k, best resolution I've ever seen it at (was on a phone and a old tablet previously). SO happy!

![zwift-4k](https://github.com/netbrain/zwift/assets/131287759/620af520-f853-421a-8e9b-fef90e1ed8bb)

Hardware:
Dell Optiplex 7060
i7-8700 
64GB RAM
using onboard GPU

Ubuntu 22.04 with docker from docker repo

**Comment by theweez510** [comment]
Do your climb graphics look really blocky? I put it on my gaming PC running Nobara 40 KDE with a 55" 4k TV and I have to scale to 275% to be able to read anything on the screen. When I run x11 scaled by system I get a proper sized mouse cursor but my climb graphics are very blocky looking, when I do scaled by apps the cursor is tiny but the climb graphic looks normal. I don't have much experience with podman/docker so haven't started looking into too much.

Hardware
Asus B550 plus Motherboard
AMD Ryzen 7 5700x
rx 7600
32 GB ram
Nobara 40 with podman from repos (does the same with docker)

**Reply by netbrain** [reply]
Can't say I have any experience with scaling, but maybe this can help? 

https://askubuntu.com/questions/1337156/how-do-i-set-scaling-in-wine
